install.md

```diff
@@ -9,13 +9,36 @@ First, clone this repository.
 This project uses Docker to build and run the code, both frontend and backend.
 So the only requirement to run it locally is a recent version of Docker with docker-compose.
 
-#### How to run
+#### How to run (with a database dump)
 
-1. (Optional) Obtain a recent database dump with name like `ietfa.torchbox.*.gz` and place in `docker/database/` directory. Otherwise, it will start with a fresh database.
+1. Obtain a recent database dump with name like `ietfa.torchbox.*.gz` and place in `docker/database/` directory. Otherwise, it will start with a fresh database.
 2. Obtain and unarchive media files into `media/` folder.
 3. Run `docker-compose up`. It will build and start the frontend builder (`yarn run start`) and the backend (`python manage.py runserver` analog), along with a Postgresql database. The first run will take a while because the database dump needs to be restored.
 4. After the frontend compilation finishes, the website should become available at http://localhost:8081
-5. To destroy everything (i.e. start the database from scratch) run `docker-compose down`.
+5. Create a super user on **Python application** docker instance to access http://localhost:8081/admin
+```
+docker exec -ti wagtail_website_application_1 python manage.py createsuperuser
+```
+6. To destroy everything (i.e. start the database from scratch) run `docker-compose down`.
+
+#### How to run (without a database dump)
+
+1. Run `docker-compose up`. It will build and start the frontend builder (`yarn run start`) and the backend (`python manage.py runserver` analog), along with a Postgresql database. The first run will take a while because the database dump needs to be restored.
+2. After **Python application** docker instance is started, run database migrations.
+```
+docker exec -ti wagtail_website_application_1 python manage.py migrate
+docker exec -ti wagtail_website_application_1 python manage.py createcachetable
+```
+3. Create an admin user
+```
+docker exec -ti wagtail_website_application_1 python manage.py createsuperuser
+```
```

Comment:
Could the `docker exec` steps not be executed by the Dockerfile?

Reply:
`migrate` and `createcachetable` steps can be added to the `Dockerfile`.
But I think it's better to leave the `createsuperuser` as a separate step because it's an interactive command and it lets user to choose a username, email and password.

Reply:
If it's interactive, I agree that it needs to be run separately. I wonder if it would still make sense to create a default admin account as part of the Dockerfile though.

Reply:
Assuming that what you meant by **default admin account** is creating a django superuser, I think it complicates the current Dockerfile.
Because the same Dockerfile is used to deploy to the production & test servers.
Even if we configure it to run only on a non production / test instance, we are introducing a security risk where some configuration mishap is going to create a default account on a  production or test instance.

Apparently, `createsuperuser` can be run non interactively (ref: https://docs.djangoproject.com/en/3.2/ref/django-admin/#createsuperuser).
But If we want to include a default admin account, it might be a better idea to include that in a some sort of stripped database dump along with other data that we want to include?

Reply:
>  Because the same Dockerfile is used to deploy to the production & test servers.

Ah - I didn't realize this. Makes total sense then not to do this.